modules/core/context.js

```diff
@@ -219,22 +219,21 @@ export function coreContext() {
     });
   };
 
-  context.zoomToNote = (noteId, zoomTo) => {
+  context.moveToNote = (noteId, moveTo) => {
     context.loadNote(noteId, (err, result) => {
       if (err) return;
       const entity = result.data.find(e => e.id === noteId);
-      if (entity) {
-        // zoom to, used note loc
-        const note = services.osm.getNote(noteId);
-        if (zoomTo !== false) {
-          context.map().centerZoom(note.loc,15);
-        }
-        // open note layer
-        const noteLayer = context.layers().layer('notes');
-        noteLayer.enabled(true);
-        // select the note
-        context.enter(modeSelectNote(context, noteId));
+      if (!entity) return;
+      // zoom to, used note loc
+      const note = services.osm.getNote(noteId);
+      if (moveTo) {
```

Comment:
In `feature_list.js` the method is called without the second parameter. With this change, iD would not pan the map to the search result anymore, or am I overlooking something?

```suggestion
      if (moveTo !== false) {
```

Reply:
Yeah, I got the logic the wrong way 'round